### PR TITLE
Adds support for queue declare args

### PR DESCRIPTION
**Pull request by wellfreire**
Adds DeclareArgs and BindArgs to ListenConfig struct:

- Those will be used when invoking CreateConsumer, which now requires both arguments;

### CODE REVIEW

File: rabbus.go [hunk 1]
```diff
@@ -71,6 +72,11 @@ type (
 		PassiveExchange bool
 		// Queue the queue name
 		Queue string
+		// DeclareArgs is a list of arguments accepted for when declaring the queue.
+		// See https://www.rabbitmq.com/queues.html#optional-arguments for more info.
+		DeclareArgs amqp.Table
+		// BindArgs is a list of arguments accepted for when binding the exchange to the queue
+		BindArgs amqp.Table
```
Comment (wellfreire) [inline rabbus.go]: I didn't find any link for docs around which args are accepted here

Comment (vgarvardt) [reply rabbus.go]: This is just free-form map, all the values are in the RabbitMQ docs. I think we can add a builder for common cases, like queue message TTL, but in another PR.

**Comment by vgarvardt** [comment]
> which now requires both arguments

they can be `nil` actually

File: rabbus.go (hunk 1 above)
Comment (wellfreire) [reply rabbus.go]: Please, note this is the `BindArgs`. The `TTL` goes with the `DeclareArgs` right?

But sure, it's a free-map, just wanted to find some docs with examples of args that could be passed here, as for the `DeclareArgs`.

Comment (mohankumarsamachar-cloud) [reply rabbus.go:79]: 089b5727e1dae376d908aad0e67ebd575153b3b8

Comment (mohankumarsamachar-cloud) [reply rabbus.go:79]: 089b5727e1dae376d908aad0e67ebd575153b3b8

**Pull request merged**